driver/ppm_flag_helpers.h

```diff
@@ -25,6 +26,9 @@ or GPL2.txt for full copies of the license.
 #include <linux/eventpoll.h>
 #include <linux/prctl.h>
 #include "ppm.h"
+#ifdef __NR_io_memfd_create
```

Comment:
ahahhaha here we are, my fault sorry, damn copy and paste!
```suggestion
#ifdef __NR_memfd_create
```

Reply:
Ahh, yes should've tried that!!